### PR TITLE
Explicit licensing, switch to CC BY 4.0 for everything

### PR DESCRIPTION
**Pull request by betatim**
This moves the license to a separate file and switches everything to CC BY 4.0

So far I am the only contributor so I think no one else needs to consent to switching the license.

This is in response to #4. What do you think @Daniel-Mietchen?

What license the output of this proposal (if it is chosen) should use is something to discuss in the actual proposal and isn't covered by LICENSE. My preference would be to write something like:

```
Work contributed to existing open source projects will be licensed under that projects terms. When a new project is created it should use the BSD (for code) or CC BY 4.0 (training material) license.
```

Basically choose as permissive a license as possible.

### CODE REVIEW

**Comment by Daniel-Mietchen** [comment]
Looks good to me, except that the BSD license should be specified - I suggest to go for the [2-clause one](https://opensource.org/licenses/BSD-2-Clause).

**Comment by betatim** [comment]
I'll merge this. Will specify the license for the code in the proposal when we start writing it.

BSD 2 clause seems fine to me as well.
